Week05/lesson10/homework/City Changer/js/index.js

```diff
@@ -0,0 +1,30 @@
+var city = document.getElementById("#city-type").value;
```

Comment:
It's for this exact reason that I always recommend using `document.querySelector` and nothing else: if you're going to use `getElementById`, since `Id` is already in the name, you don't need the `#`, ie:

`var city = document.getElementById("city-type`).value;`

OR better yet:

`var city = document.querySelector("#city-type`).value;`